### PR TITLE
whoops link is broken, fixing it.

### PR DESCRIPTION
**Pull request by hellonearth311**
## Description
the link was broken so I had to fix it..

### CODE REVIEW

**Comment by ChickenOnYT** [comment]
not useful

**Comment by ChickenOnYT** [comment]
wait it is but dont add links randomly

File: PHEx/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Combined with [Redirector](https://github.com/Prodigy-Hacking/Redirector) to inject modified game files into Prodigy.
 
-Made of the Redirector extension, no-csp, and a little extra spice ;)
+Made of the [Redirector](https://chrome.google.com/webstore/detail/redirector/ocgpenflpmgnfapjedencafcfakcekcd?hl=en) extension, no-csp, and a little extra spice ;)
```
Comment (ChickenOnYT) [inline PHEx/README.md:5]: i dont want this link remove it and i'll merge

**Comment by PatheticMustan** [comment]
no

**Pull request closed without merging**